### ISSUE TITLE
[Bug] CivBrowser tab is not displayed

### ISSUE THREAD

**Issue by teketekeFor** [SignalFlagZ/sd-civitai-browser#25]
After the update, the Civitai browser menu is missing. It is listed under the Extensions section and the folder is also there
![WS00000264](https://github.com/SignalFlagZ/sd-webui-civbrowser/assets/147781706/b8acf71d-f3c3-4e1a-addf-d35e580fec01)

**Comment by SignalFlagZ** [comment]
Thank you for the report.
There is insufficient information to reproduce the problem.
Please check the message when starting SD webUI.

**Comment by teketekeFor** [comment]
This will happen at startup
![WS00000262](https://github.com/SignalFlagZ/sd-webui-civbrowser/assets/147781706/56d892ce-c59b-4b68-a3a2-6649893980ce)

**Comment by SignalFlagZ** [comment]
Update your SD web UI >= v1.6.0-RC.

**Comment by teketekeFor** [comment]
I am using version 1.5.2 because in my environment, version 1.6 is slow to generate, so I don't use it

**Comment by teketekeFor** [comment]
I tried starting with version 1.6, but it didn't work out due to the situation described below
![WS00000265](https://github.com/SignalFlagZ/sd-webui-civbrowser/assets/147781706/1c2a23c3-be54-4c98-a260-35dcf83017f8)

**Comment by SignalFlagZ** [comment]
It's a question of how to use the SD web UI. This is not the place to answer that question.

**Comment by teketekeFor** [comment]
Until now, I was able to use the Civitai browser without any issues on version 1.5.2. I am asking this question because I've been encountering this problem since updating the extension. I am reporting that a problem has occurred. I've posted images showing the difference when using the --civsfz-api-key command and when not using it. I am not asking about how to use it.

You instructed me to use version 1.6, so the last image I posted shows the screen that appears when I launch the web UI with the command specified for version 1.6. The issue does not occur if I remove the command. I have never asked about how to use the web UI.

If version 1.5.2 does not support it, then I intend to switch to version 1.6. However, I am curious whether there is any relation between the Civtai download assisting extension and version 1.6.

*The first and second images are from right after the update. The last image I posted is the startup image when using the --civsfz-api-key in version 1.6. There are no other error codes. The only thing that appears is this list, and at the end, it says 'Press enter to continue' and that's it
